source/preprocessor.tex

```diff
@@ -876,16 +877,18 @@
 if the macro name is not lexically identical to a keyword\iref{lex.key}
 or to the \grammarterm{identifier}{s} \tcode{module} or \tcode{import},
 the first point
-of macro import of a translation unit containing a point of definition for the
-macro definition, if any (in any other translation unit).
+of macro import of a header unit containing a point of definition for the
+macro definition, if any.
+In the latter case, the macro is said
+to be \defnx{imported}{macro!import} from the header unit.
```

Comment:
I wonder if we can make this a little clearer by separating the two cases with bullets and moving the parenthesized text to the front:

> The *point of definition* of a macro definition within a translation unit <ins>T</ins> is:
>  * <ins>if the `#define` directive of the macro definition occurs within T,</ins> the point at which <del>its `#define`</del> <ins>that</ins> directive occurs <del>(in the translation unit containing the `#define` directive)</del>, or <ins>otherwise</ins>,
>  * if the macro name is not lexically identical to a keyword or to the *identifier*s `module` or `import`, the first point of macro import <ins>in T</ins> of a <del>translation</del> <ins>header</ins> unit containing a point of definition for the macro definition, if any.

Reply:
fixed